aep/general/0205/aep.md.j2

```diff
@@ -1,5 +1,47 @@
-# Beta-blocking changes
+# beta-blocking changes
```

Comment:
Why this change? Seems odd for an h1 to not start with a capital.

Reply:
Wasn't deliberate; fixed.